jujugui/static/gui/src/test/index.html

```diff
@@ -186,8 +208,8 @@
   <script src="test_simulator.js"></script>
 
   <!-- FIXME: feature flags depend on code loaded by the startup test. -->
```

Comment:
This is still the case - I have created a card as a followup as we need to find a better way to do the testing of the js in the index file.